### ISSUE TITLE
Model cannot to train

### ISSUE THREAD

**Issue by NST666** [BestAnHongjun/SentenceVAE#3]
Due to device limitations, I attempted to train SVAE with a small amount of data on a graphics card and check for overfitting. I took 1500 samples and trained 12000 batches, but although the loss value of the training was already very small, the PPL of the model was still very high and could not reach the level of the paper. Even when tested on the training set, it was impossible to effectively reconstruct. Can anyone successfully reproduce it？
